### PR TITLE
Refactor Analytics code

### PR DESCRIPTION
**Pull request by andreban**
- Create analytics.js and move analytics code there.
- Create a trackPageView method, that will be used when pages
  are changed client-side from the router.
- Change CSP header to allow importing the GA script from HTML.
- Add GA script to footer as async.

### CODE REVIEW

File: views/includes/footer.hbs [hunk 1]
```diff
@@ -28,3 +28,4 @@
   </a>
   </ul>
 <footer>
+<script async src="https://www.google-analytics.com/analytics.js"></script>
```
Comment (sebastianbenz) [inline views/includes/footer.hbs]: Would it make sense to create this programmatically in analytics.js to have it fully encapsulated?

File: public/js/analytics.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-env browser */
+
+export default class Analytics {
+  constructor(navigator, window, config) {
```
Comment (sebastianbenz) [inline public/js/analytics.js]: Why navigator as a separate parameter? You could also use window.navigator?

Comment (andreban) [inline public/js/analytics.js]: Good point. I totally forgot we could do `window.navigator`

File: views/includes/footer.hbs (hunk 1 above)
Comment (andreban) [inline views/includes/footer.hbs]: I had the same thoughts, but found that most examples available include the analytics script directly on the HTML, so decided to go with this solution for now. We may experiment with moving it to analytics.js in the future.

Comment (juliantoledo) [inline views/includes/footer.hbs]: sounds good, we should find a good way to load few of the scripts we have.

**Pull request merged**